packages/js/src/ui/components/primitives/Popover/PopoverContent.tsx

```diff
@@ -9,7 +9,7 @@ import { usePopover } from './PopoverRoot';
 export const popoverContentVariants = () =>
   cn(
     'nt-w-[400px] nt-h-[600px] nt-rounded-xl nt-bg-background',
-    'nt-shadow-popover nt-animate-in nt-slide-in-from-top-2 nt-fade-in nt-z-10 nt-cursor-default nt-flex nt-flex-col nt-overflow-hidden nt-border nt-border-border'
+    'nt-shadow-popover nt-animate-in nt-slide-in-from-top-2 nt-fade-in nt-z-[9999] nt-cursor-default nt-flex nt-flex-col nt-overflow-hidden nt-border nt-border-border'
```

Comment:
@desiprisg isn't that a bit too much, meaning the Inbox will go over a modal?

Reply:
This would be our custom popover. If it's open, i don't see why a modal would be open at the same time. However, we need to have a value that will surely be above any headers/sidebars an app might have.

Reply:
Any guess is arbitrary cause the z-index is arbitrary. I wonder if we can find a heuristic to detect the header container and always do z-index of that container + 10.

Reply:
why do we have to increase z-index? shouldn't it be always on the top of everything rn?
like the way it's rendered in the dom tree is that it's always rendered at the bottom of the body element

Reply:
Devs might have a z-index set on their elements. it doesn't work this way.

Reply:
If we can create an independent [stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_positioned_layout/Stacking_context) from the root then we won't need to bother about it. But I wasn't able to do that quickly, played here: https://codepen.io/LetItRock/pen/emYjRWX